apps/webapp/app/services/email.server.ts

```diff
@@ -14,6 +15,11 @@ const client = new EmailClient({
 });
 
 export async function sendMagicLinkEmail(options: SendEmailOptions<AuthUser>): Promise<void> {
+  // Auto redirect when in test mode
+  if (env.APP_ENV === "test") {
```

Comment:
Yea this makes a lot of sense. I think we should just do this when NODE_ENV=development

Reply:
I've updated this to auto-redirect based on `NODE_ENV === "development"`